src/zzz_od/gui/view/one_dragon/charge_plan_interface.py

```
@@ -321,6 +321,13 @@ def get_content_widget(self) -> QWidget:
         self.skip_plan_opt = SwitchSettingCard(icon=FluentIcon.FLAG, title='跳过计划', content='开启后，当前计划因体力不足或次数限制无法继续时，会依次尝试后续计划')
         self.content_widget.add_widget(HorizontalSettingCardGroup([self.loop_opt, self.skip_plan_opt], spacing=6))
 
+        self.auto_exchange_ether_battery_opt = SwitchSettingCard(
+            icon=FluentIcon.SYNC,
+            title='自动兑换以太电池',
+            content='开启后，体力计划启动时会尝试消耗储值电卡，将储蓄电量兑换为以太电池；失败时跳过并继续体力计划',
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)